http/api/transport.go

```diff
@@ -96,7 +98,9 @@ func parseSubtopic(subtopic string) (string, error) {
 }
 
 func decodeRequest(ctx context.Context, r *http.Request) (interface{}, error) {
-	if !strings.Contains(r.Header.Get("Content-Type"), contentType) {
+	if !strings.Contains(r.Header.Get("Content-Type"), contentTypeSenMLJSON) &&
+		!strings.Contains(r.Header.Get("Content-Type"), contentTypeSenMLCBOR) &&
+		!strings.Contains(r.Header.Get("Content-Type"), contentTypeJSON) {
```

Comment:
Extract to a variable `ct := r.Header.Get("Content-Type")`. Also, why don't we compare with `==` rather than using `strings.Contains`?